src/addressResolvers/unstoppableDomains.ts

```diff
@@ -13,7 +13,7 @@ const NETWORK = '137';
 const provider = getProvider(NETWORK);
 
 function normalizeHandles(handles: Handle[]): Handle[] {
-  return handles.map(h => (h.match(RegExp('^[.a-z0-9-]+$')) ? h : ''));
+  return handles.map(h => (/^[.a-z0-9-]+$/.test(h) ? h : ''));
 }
```

Comment:
Could we replace this with `normalizeHandles` in src/addressResolvers/utils.ts

Reply:
The one in utils is more global and lax, and is only filtering out invalid values like empty string, string with spaces, etc ...

The one inside each resolver are more tailored to the resolver, as they all have their own requirements 

- lens handle between 5 and 31 chars, [a-z0-0-_]
- ens allow emoji
- unstoppableDomains only allow [a-z0-0] and dash (no _)

Reply:
makes sense 👍